sdk/dotnet/Cluster.cs

```diff
@@ -592,6 +592,15 @@ public InputList<Inputs.RoleMappingArgs> RoleMappings
         [Input("skipDefaultNodeGroup")]
         public bool? SkipDefaultNodeGroup { get; set; }
 
+        /// <summary>
+        /// If this toggle is set to true, the EKS cluster will be created without the default node and cluster security groups. Instead it will use the security group created by the EKS service.
+        /// Defaults to false.
+        /// 
+        /// See for more details: https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.html.
+        /// </summary>
+        [Input("skipDefaultSecurityGroups")]
+        public bool? SkipDefaultSecurityGroups { get; set; }
```

Comment:
Weird three-state boolean. true, false, and unknown. Alas, not worth fixing probably.